py3status/modules/xscreensaver.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
There should be a single line at the top of the docstring briefly explaining the modules purpose followed by a blank line